src/os/config/displaysettings.js

```diff
@@ -55,7 +55,8 @@ os.config.DisplaySetting = {
   ENABLE_SKY: os.config.DisplaySettings.BASE_KEY + 'enableSky',
   ENABLE_LIGHTING: os.config.DisplaySettings.BASE_KEY + 'enableLighting',
   ENABLE_TERRAIN: os.config.DisplaySettings.BASE_KEY + 'enableTerrain',
-  TERRAIN_OPTIONS: os.config.DisplaySettings.BASE_KEY + 'terrainOptions'
+  TERRAIN_OPTIONS: os.config.DisplaySettings.BASE_KEY + 'terrainOptions',
+  DUSK_MODE: os.config.DisplaySettings.BASE_KEY + 'duskMode'
```

Comment:
This is a plugin feature (`plugin.x`) that is being added and referenced in a core class (`os.x`). It is better and more modular to have the plugin create and register a settings UI.